src/main/java/app/studnicki/ox/Player.java

```diff
@@ -2,7 +2,7 @@
 
 class Player {
 
-  String name;
+  final String name;
```

Comment:
Cool that final, but why not private? Just wondering.

Reply:
I use that in test class for Player.